### ISSUE TITLE
Questions about the hyper-parameter generating CLAHE real images

### ISSUE THREAD

**Issue by HuaYuuu** [zychen-ustc/PSD-Principled-Synthetic-to-Real-Dehazing-Guided-by-Physical-Priors#33]
Hello, Thanks for your incredible work! Now I'd like to run the PSD dehazing method on my own dataset. However, the fine-tuning step requires the CLAHE version of real haze images.  I have downloaded the CLAHE code (Matlab version), but I'm not sure how to set the parameters in CLAHE. I'd appreciate it if you could offer some advice about how to set this hyper-parameter. Thanks!

p.s.
CLAHE code (Matlab Version)https://ww2.mathworks.cn/matlabcentral/fileexchange/22182-contrast-limited-adaptive-histogram-equalization-clahe
[CEImage] = runCLAHE(Image,XRes,YRes,Min,Max,NrX,NrY,NrBins,Cliplimit);
%    Image - The input/output image
%    XRes - Image resolution in the X direction
%    YRes - Image resolution in the Y direction
%    Min - Minimum greyvalue of input image (also becomes minimum of output image)
%    Max - Maximum greyvalue of input image (also becomes maximum of output image)
%    NrX - Number of contextial regions in the X direction (min 2, max uiMAX_REG_X)
%    NrY - Number of contextial regions in the Y direction (min 2, max uiMAX_REG_Y)
%    NrBins - Number of greybins for histogram ("dynamic range")
%    Cliplimit - Normalized cliplimit (higher values give more contrast)
